web/modules/custom/hello_user/hello_user.permissions.yml

```diff
@@ -0,0 +1,3 @@
+Hello User Permission:
```

Comment:
where is this used?

Reply:
It is used in the routing file: hello_user.routing.yml